lms/urls.py

```diff
@@ -96,6 +96,7 @@
     url(r'', include('openedx.features.specializations.urls')),
     url(r'^marketplace/', include('openedx.features.marketplace.urls')),
     url(r'^idea/', include('openedx.features.idea.urls')),
+    url(r'^smart_referral/', include('openedx.features.smart_referral.urls')),
```

Comment:
Let's change URL from `url(r'^smart_referral/', include('openedx.features.smart_referral.urls')), ` to `url(r'^referral/', include('openedx.features.smart_referral.urls')),`